packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -196,8 +196,11 @@ export class LionInputDatepicker extends LionInputDate {
    */
   _requestUpdate(name, oldValue) {
     super._requestUpdate(name, oldValue);
+
     if (name === 'disabled') {
       this.__delegateDisabled();
```

Comment:
would be nice to have a demo in the storybook for `Readonly` in addition to `Disabled` that we currently have

I checked how they would look like and probably smth goes wrong with the current implementation

![image](https://user-images.githubusercontent.com/137844/59852265-c3130500-936e-11e9-9302-7b0ac1578d78.png)

Reply:
Good idea. Will do

Reply:
I found out why there is a difference between how disabled and readonly buttons looks like

for disabled we have this style:

```css
            :host(.state-disabled) .input-group ::slotted(*) {
              color: var(--disabled-text-color, #adadad);
            }
```

not for readonly though

this is just a side-effect of curse, there was no intention to make a different colour of the button, but it is a bug, because in both cases the button should look like disabled and in both cases we need consistent styles which we just don't define and face a strange inconsistent side effect

Reply:
I will create another but report after we finish this